dojo/urls.py

```diff
@@ -150,7 +152,17 @@
     url(r'^schema/$', SchemaView.as_view(), name='schema'),
 ]
 
-schema_view = get_swagger_view(title='Defect Dojo API v2')
+schema_view = get_schema_view(
+    openapi.Info(
+        title="Defect Dojo API",
+        default_version='v2',
+        description="To use the API you need be authorized.",
+    ),
+    # if public=False, includes only endpoints the current user has access to
+    public=True,
+    # The API of a OpenSource project should be public accessible
```

Comment:
Does this really apply with defect Dojo @aaronweaver the previous doc used did not reveal all endpoint to unauthenticated users...

Reply:
If we want to restrict the api to authenticated users only, we can do this with the permission classes (line 164). The public flag configures only if a user with access can see the hole api documentation or only functions he can use. 

From my point of view the api access is already restricted and we do not need to restrict the access to the documentation.

Reply:
I believe what @propersam was trying to say is in an instance of multiple products with varying users having access. If a user does not have access to a given product, they are not allowed to see any endpoints or findings. However, these endpoints could still be seen through API usage. Simply restricting by authenticated users would not completely satisfy this situation.

Reply:
We have to distinguish between the authentication for the api and the authentication for the api documentation. This PR is only about the api documentation and does not change the authentication for the api.

Reply:
I think I understand now. What you're talking about is the auth to access the api whereas Sam  and I were talking about the results from using the api. Is that correct?

Reply:
I was actually referring to d authentication on d api docs... @alles-klar reasons for making the api docs open..seems like a valid reason..it does nt affect d user authentication....unauthenticated users won't be able to use any endpoint that requires authentication but they would be allowed to see it exists frm d doc

Reply:
correct

Reply:
V1 docs are already this way so now it's consistent I guess. Null auth error messages pop correctly too. I don't see a problem with this anymore. Sorry for the confusion!